fairlearn/metrics/metric_result.py

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+
+class MetricResult:
+    def __init__(self):
```

Comment:
GroupMetricResult?

Reply:
I'm a bit concerned with the proliferation of small files.

Reply:
Changed to GroupMetricResult